### ISSUE TITLE
 increased glycogen in myocardium 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9595]
**Preferred term label:**
 increased glycogen in myocardium 

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
261740 
PMID: 15877279  5/5

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Increased myocardial glycogen content

added
